src/templates/kit/screens/FundraisingScreen.js

```diff
@@ -270,19 +177,53 @@ function FundraisingScreen({
     return true
   }, [fields, minimumGrowth])
 
+  const fieldsAreValid = useMemo(() => {
+    const error = validationError(fields)
+    setFormError(error)
+
+    if (minimumGrowth.gte(fields.expectedGrowth)) {
+      return false
+    }
+
+    if (fields.fundingPeriod >= fields.cliffPeriod) {
+      return false
+    }
+
+    if (fields.cliffPeriod >= fields.vestingSchedule) {
+      return false
+    }
+
+    return true
+  }, [fields, minimumGrowth])
+
+  function validationError(fields) {
+    if (fields.fundingPeriod >= fields.cliffPeriod) {
```

Comment:
Just curious; do we also need to handle an error for `minimumGrowth.gte(fields.expectedGrowth)`?